addons/binding/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/handler/FreeboxHandler.java

```diff
@@ -175,15 +189,15 @@ private boolean authorize(boolean useHttps, String fqdn, String apiBaseUrl, Stri
             loginManager.setAppToken(configuration.appToken);
             loginManager.openSession();
             return true;
-        } catch (FreeboxException | InterruptedException e) {
-            logger.error("{}", e.getMessage());
+        } catch (Exception e) {
+            logger.warn("Thing {}: error while opening a session", getThing().getUID(), e);
```

Comment:
I changed to logger.debug. Thing status is updated to OFFLINE and CONFIGURATION_ERROR by caller of this method (when it returns false).